homeassistant/components/sensibo/strings.json

```diff
@@ -31,6 +31,28 @@
     }
   },
   "entity": {
+    "switch": {
+      "_": {
```

Comment:
This is not a entity component, this still needs to be adjusted as per previous review comment.

Reply:
Then I probably still don't understand how the syntax should be to translate attributes for various entities.
I know Sensibo isn't making any entity components and would prefer to skip the translation key if possible as the attributes partially share same names but maybe that's not possible (or at least not short term).
Could you provide me with an example perhaps?

Reply:
> and would prefer to skip the translation key if possible

That is not possible / allowed.

I've linked examples before above, for example, the thermostat of the plugwise integration. Another example can be found in the `moon` integration (which is a single sensor).

Reply:
> That is not possible / allowed.

That was sort of the missing link. I would have hoped to make "generic" translations instead of using a translation key in in general.
But anyway this should be sorted now according to guidelines so appreciate a look at this one to close it.
Or maybe merge #85234 first and then we can use some more keys instead.